tests/client/core/api/test_api.js

```diff
@@ -7,18 +7,26 @@ import { ErrorHandler } from 'core/errorHandler';
 import { unexpectedSuccess } from 'tests/client/helpers';
 
 
+const defaultHeaders = {
+  'Content-Type': 'application/json',
+};
+
+function generateHeaders(headerData = defaultHeaders) {
```

Comment:
I would find this slightly more easy to read if it were all in the same place:
````js
function generateHeaders(
  headerData = { 'Content-Type': 'application/json' }
) {
  ...
}
````